lib/elixir_internal_certificate/scrapper/scrappers.ex

```diff
@@ -0,0 +1,32 @@
+defmodule ElixirInternalCertificate.Scrapper.Scrappers do
```

Comment:
As a side note, it should be `scraper`. With two `pp`s it corresponds to another word:

![image](https://user-images.githubusercontent.com/696529/175933738-fe11fca9-451a-4c9c-9d74-08eaf3f56308.png)

😭

Reply:
@olivierobert I didn't realize that 🥲 I'm fixing them now